src/components/DigitalCredentials/CredentialsLanding.vue

```diff
@@ -61,6 +64,9 @@ import { ImportantMessage } from '@/components/common'
   }
 })
 export default class CredentialsLanding extends Vue {
+  @Prop({ default: [] })
+  readonly issuedCredentials: Array<DigitalCredentialsIF>
```

Comment:
Please make this 1 line and add the Typescript null assertion character.

Reply:
Updated!